### DISCUSSION TITLE
EJBCA Rest API link not auto generating when we use EJBCA Community Edition in Docker

### DISCUSSION THREAD

**Discussion by libinmath3w** [Keyfactor/ejbca-ce#178] [Q&A]
Hi All,

i want to generate TLS and other certificates using api. However, the documentation mentioned below lines which are not showing my ejbca server.

>  Each EJBCA release is bundled with an auto-generated REST API reference available on the URL https://localhost:8442/ejbca/doc/rest/ 
>     if deployed.

if i visit example.com/ejbca/doc/rest/ then i get 404 error page. i have already enabled api in protocol section. how can i fix this issues.

![Screenshot (412)](https://user-images.githubusercontent.com/24878821/204728072-366e993d-8525-4f66-8d9d-2ba1a2c51a7f.png)

**Comment by primetomas** [comment]
See #177 for an answer/link.
